docs/docs/api/Tasks/fetchTaskByDay.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Fetch Task By Day
+---
+
+export const Endpoint = ({children, color}) => ( <span style={{
+borderRadius: '2px',
+color: '#E83E8C',
+}}>{children}</span> );
+
+<Endpoint>GET tasks/day/:day/</Endpoint>: Fetching a single Task
```

Comment:
Minor comment: Similar to the other comments on the other pages, please add the `/` on your endpoints (for consistency)

Reply:
Fixed: Changes are reflected in recent commits